src/model/control_desk.py

```diff
@@ -127,53 +137,69 @@ def encoder_position(self, position: int):
 class ColorDeskColumn(DeskColumn):
     def __init__(self, _id: str = None):
         super().__init__(_id)
-        self._color_h = 0.0
-        self._color_s = 0.0
-        self._color_i = 0.0
+        self.color: ColorHSI = ColorHSI(0, 0, 0)
 
     def _generate_column_message(self) -> proto.Console_pb2.fader_column:
         base_msg = self._generate_base_column_message()
-        base_msg.plain_color.hue = self._color_h
-        base_msg.plain_color.saturation = self._color_s
-        base_msg.plain_color.intensity = self._color_i
+        base_msg.plain_color.hue = self.color.hue
+        base_msg.plain_color.saturation = self.color.saturation
+        base_msg.plain_color.intensity = self.color.intensity
         return base_msg
 
     @property
-    def color(self) -> Tuple[float, float, float]:
-        return (self._color_h, self._color_s, self._color_i)
+    def color(self) -> ColorHSI:
+        """color of the colum"""
+        return self.color
 
     @color.setter
-    def color(self, h: float, s: float, i: float):
-        self._color_h = h
-        self._color_s = s
-        self._color_i = i
+    def color(self, color: ColorHSI):
+        self.color = color
         self.update()
 
 
 class FaderBank:
+    """TODO Docstring"""
```

Comment:
"""Store a bank page of columns the user can switch through.

Warning: As of the time of this writing, the buttons for scrolling through a bank are not implemented in fish, hence the user can only access up to 8 columns.
"""